tasks.py

```diff
@@ -33,6 +41,11 @@ def prepare(ctx, version):
         config.write(configfile)
 
     print('Installing github-changelog')
+    # Get last modified date from Git instead of assuming the file metadata is
+    # correct. This can change depending on git reset, etc.
+    git_log = ctx.run('git log -1 --format="%ad" -- CHANGELOG.rst')
+    last_modified = parse(git_log.stdout.strip()).strftime('%Y-%m-%d')
```

Comment:
We should put a note in the changelog that it's auto-generated, and not to touch it manually.

Reply:
Actually, one of the benefits of this tool is that it only does that last release. It does allow for authoring in between generating and commiting, so we can add deployment and migration notes, or more general description of some of the changes